how-to/taskcluster/new_region.rst

```diff
@@ -0,0 +1,13 @@
+.. new_region:
+
+Adding a new region
+===================
+
+Adding a new GCP region for taskcluster workers requires changes in several places:
+
+- if the workers are going to clone the repository, add clonebundles in that region (e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=1868861)
```

Comment:
something like "if the workers are going to clone a Mercurial Gecko repository" might be more appropriate. This isn't necessary for Github, and might not (I honestly don't) be necessary for smaller hg repos).

Reply:
Thanks, updated to have these first two items in a "for gecko and comm pools" section.